mod/Jailbreak.SpecialDay/SpecialDays/OneInTheChamberDay.cs

```diff
@@ -0,0 +1,73 @@
+using CounterStrikeSharp.API;
+using CounterStrikeSharp.API.Core;
+using CounterStrikeSharp.API.Modules.Entities;
+using CounterStrikeSharp.API.Modules.Entities.Constants;
+using Jailbreak.English.SpecialDay;
+using Jailbreak.Formatting.Views;
+using Jailbreak.Public.Extensions;
+using Jailbreak.Public.Mod.SpecialDay;
+using Jailbreak.Public.Mod.SpecialDay.Enums;
+using Jailbreak.Public.Utils;
+
+namespace Jailbreak.SpecialDay.SpecialDays;
+
+public class OneInTheChamberDay(BasePlugin plugin, IServiceProvider provider) 
+  : FFADay(plugin, provider) {
+  
+  public override SDType Type => SDType.OITC;
+  
+  public override ISpecialDayInstanceMessages Messages
+    => new SoloDayMessages("One In The Chamber",
+      "You only have one bullet. Kill someone else to get another bullet. "
+      + "No camping!");
+
+  public override SpecialDaySettings Settings => new OitcSettings();
+  
+  public override void Setup() {
+    Timers[120] += () => {
+      foreach (var player in PlayerUtil.GetAlive()) {
+        player.GiveNamedItem("weapon_knife");
+      }
+    };
+    plugin.RegisterEventHandler<EventPlayerHurt>(OnPlayerDamage);
+    plugin.RegisterEventHandler<EventPlayerDeath>(OnPlayerDeath);
+    base.Setup();
+  }
+  
+  public override void Execute() {
+    foreach (var player in PlayerUtil.GetAlive()) {
+      player.RemoveWeapons();
+      player.GiveNamedItem(CsItem.Deagle);
+      player.GetWeaponBase("weapon_deagle").SetAmmo(1, 0);
+    }
+    base.Execute();
+  }
+
+  private HookResult OnPlayerDamage(EventPlayerHurt @event, GameEventInfo info) 
+  {
+    var  player     = @event.Userid!;
+    bool usedDeagle = @event.Weapon.Contains("deagle");
+    if (!player.IsValid || !usedDeagle) return HookResult.Continue;
+    
+    player.PlayerPawn.Value!.Health = 0;
```

Comment:
Can we just call player.Suicide? Alternatively we might look into modifying the event's damage if that's possible.

Either way, should use our PlayerExtensions#SetHealth to avoid having to update the state "manually"

Reply:
I don't wanna suicide cause i wanna maintain the kill logs showing who killed who, and as far as i know it's not possible to directly edit the event's damage. I switched it to use the player extension.